other/bootstrap_daemon/README.md

```diff
@@ -50,23 +48,55 @@ Get your public key and check that the daemon initialized correctly:
 sudo grep "tox-bootstrapd" /var/log/syslog
 ```
 
+##For `systemd` users:
+
+You must uncomment the next line in tox_bootstrap.service, if you want to use port number <1024 
+	
+	#CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+
+Place tox_bootstrap.service into /etc/systemd/system:
+```sh
+sudo cp tox_bootstrap.service into /etc/systemd/system/
+```
+
 
+Reload systemd units definitions and start service: 
+```sh
+sudo systemctl daemon-reload
+sudo systemctl start tox_bootstrap.service
```

Comment:
You should also add `sudo systemctl enable tox-bootstrapd.service`. Doesn't seem to work for me without that, the daemon doesn't automatically start after reboot.